ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -16,6 +16,7 @@ export interface CardHeader {
   content: string | React.ReactNode
   href?: string
   customClass?: string
+  style?: AppearanceStyleType
```

Comment:
We're in the process of refactoring from `style` to `styleType` in a few places so as to avoid conceptual conflicts with the HTML `style` attribute, so let's change this to `styleType`.